services/checks/checks.go

```diff
@@ -64,6 +69,8 @@ const (
 	scriptExecutionTimeout = 5 * time.Second  // time limit for running pmm-managed-starlark
 	resultCheckInterval    = time.Second
 
+	pmmVictoriaMetricsAddress = "http://127.0.0.1:9090/prometheus"
```

Comment:
we have victoriaMetricsURLF in main.go, let's reuse it in case we extract VM outside of PMM in future.

Reply:
Done